scenario/execute.py

```diff
@@ -0,0 +1,39 @@
+from edge.models import Edge
+from request.managers import RequestExecution
+from .signals import scenario_run_finished
+
+
+class Execution:
```

Comment:
esm e class be nazaram ScenarioExecution bashe

Reply:
movafegham